src/AddToCampaignHandler.php

```diff
@@ -317,6 +317,13 @@ public function addToCampaign($object, $data)
 
         $changeSet->addObject($object);
 
+        $childObjects = $object->findRelatedObjects('cascade_addtocampaign', true);
```

Comment:
```suggestion
        $childObjects = $object->findRelatedObjects('cascade_add_to_campaign');
```
Make the name of the config more consisten with other config variables.
And we don't need to pass true as the second argument, it's true by default.